src/Surface_TEST.cc

```diff
@@ -126,3 +190,242 @@ TEST(DOMcontact, CollideBitmask)
   EXPECT_EQ(contact.CollideBitmask(), 0x67);
 }
 
+/////////////////////////////////////////////////
+TEST(DOMfriction, DefaultConstruction)
```

Comment:
Since the test is named `DefaultConstruction`, I think it should test the default values, not ones set using the `Set*` functions.